src/Overview.tsx

```diff
@@ -51,15 +80,68 @@ export default function Overview() {
           battery_level: status.batteryLevel(),
           status: status.status() ?? '',
           location: status.location() ?? '',
+          is_active: true,
+          last_updated: 'now',
         },
       }));
     }, [])
   );
 
+  useEffect(() => {
+    (async () => {
+      const baseUrl = new URL(config.serverUrl);
+      baseUrl.protocol = window.location.protocol;
+      const url = new URL('robots', baseUrl).toString();
+      const res = await fetch(url, {
+        method: 'get',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+      });
+
+      let robotInfo = await res.json();
+      robotInfo.forEach((robot: StaticRobotInfo) => {
+        const name = '/' + robot.name;
```

Comment:
seems odd that you have to do this... I assume to make the URLs work or something? shouldn't the server already have a fully qualified robot name or is there something weird going on

Reply:
This is because on the server side, we store the robot name as  the namespace from which the `status` topic is drawn, which means that it doesn't include the leading slash.
So without this, if the jackal was online, we would see a table row for `jackal` as well as an "online" table row for `/jackal`.

An alternative here is to have the server store robot names with the `/` as well, I have no strong preferences